joschahenningsen/TUM-Live
# Feature/chat reactions

### Motivation and Context
Students want a quick and easy way to express the most common thoughts on messages.

### Description
Gives students the possibility to react to messages.

### Steps for Testing
Prerequisites:
- 2 Accounts
- 1 Livestream

1. Log in
2. Navigate to a Livestream
3. Write some messages
4. React to the messages from the two accounts
5. Check if reactions are synced in realtime and if they can be removed again.

### Screenshots

![Bildschirmfoto 2022-12-18 um 04 11 02](https://user-images.githubusercontent.com/17506411/208279440-fc16b313-3738-422d-a44d-0765a0f42825.png)

![Bildschirmfoto 2023-01-01 um 18 25 57](https://user-images.githubusercontent.com/17506411/210182589-af3b6c36-05df-44ee-abd3-0b2c3611ce6d.png)


https://user-images.githubusercontent.com/17506411/210182578-d7ec4fd1-ff0e-4eed-aa88-0d4516584f60.mov
